pkg/services/grafana-apiserver/endpoints/request/namespace.go

```diff
@@ -20,6 +22,22 @@ type NamespaceInfo struct {
 	Value string
 }
 
+// NamespaceMapper converts an orgID into a namespace
+type NamespaceMapper = func(orgId int64) string
+
+// GetNamespaceMapper returns a function that will convert orgIds into a consistent namespace
+func GetNamespaceMapper(cfg *setting.Cfg) NamespaceMapper {
```

Comment:
i need to push up the branch that separates the refactor from file storage, but in #75759 i moved this file into a separate package so that k8s related code/utils that the SDK might eventually use wouldn't get mixed with Grafana's wire dependencies. i'm not against this change if it seems like the better way to organize things

Reply:
I think the two should live together, but I don't really care where -- feel free to move elsewhere